pkg/config/config.go

```diff
@@ -121,14 +118,23 @@ func (b Blueprint) ModuleGroup(mod string) (DeploymentGroup, error) {
 }
 
 // ModuleGroupOrDie returns the group containing the module; panics if unfound
-func (b Blueprint) ModuleGroupOrDie(mod string) DeploymentGroup {
-	g, err := b.ModuleGroup(mod)
+func (bp Blueprint) ModuleGroupOrDie(mod string) DeploymentGroup {
+	g, err := bp.ModuleGroup(mod)
 	if err != nil {
 		panic(fmt.Errorf("module %s not found in blueprint: %s", mod, err))
 	}
 	return g
 }
 
+func (bp *Blueprint) groupOrder(g DeploymentGroup) int {
```

Comment:
The name `groupOrder` suggests a "is it sorted?" `bool` to me. I think this would be better as `groupIndex`.